src/python/pants/backend/python/rules/pex.py

```diff
@@ -24,10 +24,12 @@
 
 @dataclass(frozen=True)
 class PexRequirements:
-  requirements: FrozenSet[str] = frozenset()
+  requirements: Tuple[str, ...] = ()
```

Comment:
Can we make a signature for OrderedSet? Or would it not be generic anymore?

Reply:
Correct. We would have to teach TC how to use generics. Also OrderedSets aren't immutable and it would mean introducing imports in a bunch of places.

Reply:
Mm. True, yep.